public/content/developers/docs/nodes-and-clients/node-architecture/index.md

```diff
@@ -4,21 +4,23 @@ description: Introduction to how Ethereum nodes are organized.
 lang: en
 ---
 
-An Ethereum node is composed of two clients: an [execution client](/developers/docs/nodes-and-clients/#execution-clients) and a [consensus client](/developers/docs/nodes-and-clients/#consensus-clients).
+An Ethereum node is composed of a minimum of two clients: an [execution client](/developers/docs/nodes-and-clients/#execution-clients) and a [consensus client](/developers/docs/nodes-and-clients/#consensus-clients). For a node to propose a new block, it must also run a [validator client](#validators) and stake ETH.
```

Comment:
I'm not sure I agree with putting this in the intro here, as it feels off topic to me starting to talk about block proposals. The basic architecture is consensus + execution client pairing, and as you've noted the validator client software is part of the CL client and optional.

If your intent to just make this more clear to everyone reading that this other part is separate?

Reply:
It's my attempt to clarify the choices early -- you need EL and CL for one node type, for another node type validator too. Else the first contact with the term "Full node" can be packed with assumptions.

The stake ETH part is probably redundancy and can go.

WDYT?

Reply:
I suppose it just feels extraneous here, as it's not essential to running a node. But that also seems to be your point: clarify that early. I suppose I'm fine with this... try to clear up common misconceptions.

I'm still not sure I'd add "a minimum of"... I'd keep this straight forward: EL + CL pair = a "node".

Reply:
fixed minimum of